backend/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM golang:1.7.1
+FROM ubuntu:yakkety
```

Comment:
do we still need to patch this file at all since we don't need libelf, bcc, etc.?